docs/src/main/tut/middleware.md

```diff
@@ -197,19 +197,25 @@ We can create a middleware that registers metrics prefixed with a
 provided prefix like this.
 
 ```tut:silent
-import cats.effect.{Resource, IO}
+import cats.effect.{Clock, IO, Resource}
+import org.http4s.HttpRoutes
+import org.http4s.metrics.prometheus.{Prometheus, PrometheusExportService}
+import org.http4s.server.Router
 import org.http4s.server.middleware.Metrics
-import org.http4s.metrics.prometheus.Prometheus
-import io.prometheus.client.CollectorRegistry
 ```
 ```tut:book
 implicit val clock = Clock.create[IO]
 
-val meteredRoutes: Resource[IO, HttpRoutes[IO]] =
+def meteredRouter(routes: HttpRoutes[IO]): Resource[IO, HttpRoutes[IO]] =
```

Comment:
This should be called somewhere with the `apiService` now its just a function that is not used in this example.

Reply:
Hm, I guess I thought getting the reader to a Resource[IO, HttpRoutes[IO]] would be sufficient (without writing an entire sample server).

Reply:
This is better, but we should still call this method with `apiService` to show use.
or we can change this back to use the `apiService` directly

Reply:
I see what you mean. I'll make that change.